dace/properties.py

```diff
@@ -662,31 +748,6 @@ def from_string(s):
         return di
 
 
-class ParamsProperty(Property):
```

Comment:
Why is this removed? Is it deprecated by the new DictProperty?

Reply:
Yes, `ListProperty` to be precise.